runners/flink/src/test/java/org/apache/beam/runners/flink/PortableExecutionTest.java

```diff
@@ -38,10 +38,15 @@
 import org.apache.beam.sdk.transforms.DoFn;
 import org.apache.beam.sdk.transforms.GroupByKey;
 import org.apache.beam.sdk.transforms.Impulse;
+import org.apache.beam.sdk.transforms.MapElements;
```

Comment:
Can we instead run the RequiresStableInputIT instead?

https://github.com/apache/beam/blob/434427e90b55027c5944fa73de68bff4f9a4e8fe/sdks/java/core/src/test/java/org/apache/beam/sdk/RequiresStableInputIT.java

Reply:
Fixed and used the IT test for both classic and portable runners.